src/device-registry/utils/insert-measurements.js

```diff
@@ -52,19 +52,19 @@ const insert = async (res, tenant, transformedMeasurements) => {
     }
   }
   if (errors.length > 0) {
-    return res.status(HTTPStatus.BAD_GATEWAY).json({
+    return {
       success: false,
       message: "finished the operation with some errors",
       errors: errors,
       valuesRejected: eventsRejected,
       valuesAdded: eventsAdded,
-    });
+    };
```

Comment:
bravo @Baalmart

Reply:
@Baalmart , seems to get events work perfectly, however when I specified startTime=2021-04-14, get results for 15th April included, however when I specify 2021-04-14, results for  15th April are not displayed

Reply:
Thanks @sserurich :

Could you shed some light on the second specification of "2021-04-14", could that be the `endTime` query parameter?

- Please note that just specifying the startTime brings all values from the start time onwards up to present. With a default of 50 records by the system.
- Specifying the same startDate and endTime brings the values for that day only.....